src/test/kotlin/minesweeper/domain/MineGeneratorTest.kt

```diff
@@ -0,0 +1,17 @@
+package minesweeper.domain
+
+import io.kotest.core.spec.style.BehaviorSpec
+import io.kotest.matchers.shouldBe
+
+class MineGeneratorTest : BehaviorSpec({
+    Given("지뢰 개수가 주어지면") {
+        val count = 10
```

Comment:
```suggestion
        val count = 10
        val width = 10
        val height = 10
```
width, height 도 주어지면 좋을거같아요 😄

Reply:
수정하겠습니다!